lib/EthEventsClient.js

```diff
@@ -97,6 +97,35 @@ export default class EthEventsClient {
         }
     }
 
+    async getLatestBlock() {
+        const response = await axios.post(`${this._baseUrl}/block/search/`, {
+            'query': {
+                'match_all': {}
+            },
+            'size': 1,
+            'sort': [
+                {
+                    'number.num': {
+                        'order': 'desc'
+                    }
+                }
+            ],
+            '_source': [
+                'hash',
+                'number.num'
+            ]
+        }, this._axisConfig)
+        if (response.data.hits.hits.length) {
```

Comment:
In theory this might lead to an exception and you could check if every level is available like
```response && response.data && response.data.hits ... && response.data.hits.hits.length```

Reply:
But this might show a strange behaviour in elastic. I'd let it be like that as the error is catched in the controller and thus lead to the same error code.